src/pages/software-engineering/version-control-system/index.md

```diff
@@ -3,13 +3,24 @@ title: Version Control System
 ---
 ## Version Control System
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/software-engineering/version-control-system/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+A Version Control System is a tool used by developers to manage changes to files, documents and computer programs over time. 
```

Comment:
I would remove the word `developers`. Version control is used by all sorts of people, it can be useful when writing an academic thesis for example.